blockdevice/stats.go

```diff
@@ -86,11 +87,100 @@ type Diskstats struct {
 	IoStatsCount int
 }
 
+// BlockQueueStats models the queue files that are located in the sysfs tree for each block device.
```

Comment:
Is there any online documentation we can link to for details about these fields?